src/System.Runtime.CompilerServices.Unsafe/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+
+  <Import Project="..\Directory.Build.props" />

```

Comment:
The SDK team recommends to use the GetPathOfFileAbove function so that you don't need to resolve the path to the next outer D.B.props file manually or need an intermediate (empty) Directory.Build.props file:

```xml
<Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props, $(MSBuildThisFileDirectory)..))" />
```

Reply:
@ericstj Isn't this the one-liner you were talking about?

Reply:
Yep, exactly.